src/qibolab/oneQ_emulator.py

```diff
@@ -0,0 +1,56 @@
+from qibolab.emulator import create_runcard_emulator
+from qibolab.instruments.simulator.models import general_no_coupler_model
+from qibolab.instruments.simulator.pulse_simulator import (
+    PulseSimulator,
+    get_default_simulation_config,
+)
+
+# device parameters: frequency in Hz, time in s
+device_name = "ibmfakebelemQ0"
+sampling_rate = 4500000000.0
+readout_error = [0.01, 0.02]
+lo_freq = 5090167234.445013
+anharmonicity = -336123005.1821652
+drive_freq = 5090167234.445013
+rabi_freq = 125457538.19061986
+T1 = 8.857848970762537e-05
+T2 = 0.00010679794866226273
+nlevel = 3
+
+# simulation parameters
+sim_sampling_boost = 10
+simulate_dissipation = True
+instant_measurement = True
+
+
+def create_oneQ_emulator(runcard_folder: str):
+    """Create a one qubit emulator platform."""
+
+    model_params_dict = {
+        "device_name": device_name,
+        "topology": [],
+        "nqubits": 1,
+        "ncouplers": 0,
+        "qubits_list": ["0"],
+        "couplers_list": [],
+        "sampling_rate": sampling_rate,
+        "readout_error": {0: readout_error},
+        "drive_freq": {"0": drive_freq},
+        "T1": {"0": T1},
+        "T2": {"0": T2},
+        "lo_freq": {"0": lo_freq},
+        "rabi_freq": {"0": rabi_freq},
+        "anharmonicity": {"0": anharmonicity},
+        "coupling_strength": {},
+    }
```

Comment:
Eventually, this could be directly a section in the `model.json` file, so you won't need to collect the parameters here.

Reply:
So for the one qubit emulator, I am trying out this beginner/user friendlier version where the user can directly modify any of the relatively short list of required parameters, which is a subset of the full `model_params_dict`. Otherwise, this is in fact the dictionary that appears in `model.json`, but the user will be exposed to the other entries as well. If this is not useful, I can simply revert it to the typical emulator setup as per `emulator.py` which would load the full `model_params_dict` from `model.json`.

Reply:
Fixed in 76d5cb5 and 15307d6